docs/src/main/asciidoc/remove-credentials-from-spring-kafka-applications.adoc

```diff
@@ -9,14 +9,14 @@ You can use the Event Hubs Kafka endpoint in your Spring Kafka application. From
 
 First, add the `spring-cloud-azure-dependencies` BOM, as shown in the following example:
 
-[source,xml]
+[source,xml,indent=0,subs="attributes,verbatim"]
 ----
 <dependencyManagement>
   <dependencies>
     <dependency>
       <groupId>com.azure.spring</groupId>
       <artifactId>spring-cloud-azure-dependencies</artifactId>
-      <version>${version.spring.cloud.azure}</version><!--	The version for spring-cloud-azure-dependencies is 4.3.0+.-->
+      <version>{project-version}</version>
```

Comment:
I remember this will not be replaced in the generated html. Please have a try.

Reply:
Tested, this will be simpler.